### ISSUE TITLE
Still no results

### ISSUE THREAD

**Issue by duncancrozier** [jamalneufeld/ghostHunter#60]
#57 & #11 

Hi, I'm still getting no results through. I've just updated my Ghost CLI and upgraded to v.1.23.0 (current latest). No JS errors and ghostHunter: init from the console log.

I think I'm at the limit of my knowledge here, so any pointers would be appreciated.
cheers.

**Comment by duncancrozier** [comment]
Hi Windyo - thanks for the quick reply.
I rolled back to v.0.4.0 and it's working fine again... really not sure on what went wrong with v.0.5.0 but I'll leave it on the old version for the time being.
However, the {{PRETTYPUBDATE}} is displaying as that it's not actually rendering out the publish date... I may have read something previously about an old issue around this but can you recall a fix for this or I can always declare a custom result_template.
Cheers

**Comment by duncancrozier** [comment]
Changing {{prettyPubDate}} to {{pubDate}} has solved it ;)

**Comment by Windyo** [comment]
Yeah .4 has a thing where prettypubdate works for me but for no one else. I've never been able to diagnose that one because I can't reproduce it...

As far as 0.4 working and 0.5 not, we changed a couple of functions that are called by jQuery. Mind checking which version of jQuery you're including in you blog ? It's possible the issue comes from there.

**Comment by duncancrozier** [comment]
When I updated the ghost cli and updated to the latest version of ghost I also updated the jquery version to 1.12.4.

**Comment by Windyo** [comment]
Now I'm actually curious.

I'll try to test this again. Too many people ahving issues with 5.0 for it to be a coincidence

**Comment by max-nova** [comment]
I suspect it may be the .size() call in line 347 of ghostHunter.js?  .size() has been deprecated since jQuery 1.8 in favor of .length (https://api.jquery.com/size/)

Was having trouble getting ghostHunter to work until I changed that line.
